force-app/main/default/chat/class/controller/ChatController.cls

```diff
@@ -0,0 +1,14 @@
+public with sharing class ChatController {
+    public static IChatService chatService;
+
+    static {
```

Comment:
Why do you use a static block here?

Reply:
deleted unnecessary static block force-app/main/default/chat/class/controller/ChatController.cls